src/constants.js

```diff
@@ -9,19 +9,24 @@ const GITHUB_REPOSITORY = process.env["GITHUB_REPOSITORY"];
 
 const REPO_DIRECTORY = process.env["GITHUB_WORKSPACE"];
 
+const JPEG_QUALITY = process.env["INPUT_JPEGQUALITY"] || 80;
+const PNG_QUALITY = process.env["INPUT_PNGQUALITY"] || 80;
+const WEBP_QUALITY = process.env["INPUT_WEBPQUALITY"] || 80;
+const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"].split(",") || ["node_modules/**"];
+
 const COMMITTER = {
   name: "Calibre",
   email: "hello@calibreapp.com"
 };
 
 if (!REPO_DIRECTORY) {
-  console.log("There is no GITHUB_WORKSPACE environment variable");
+  console.log("::error::There is no GITHUB_WORKSPACE environment variable");
   process.exit(1);
-}
+};
 
 const CONFIG_PATH = path.join(
   REPO_DIRECTORY,
-  ".github/calibre/image-actions.yml"
+  process.env["INPUT_CONFIPATH"] || ".github/calibre/image-actions.yml"
```

Comment:
Seeing as the `.github/calibre/image-actions.yml` is no longer the default suggested config method, let's remove the `INPUT_CONFIGPATH` input. 

Pre-existing installations will still work, but there's no need to suggest people use the non-preferred config method.